pootle/static/js/editor/app.js

```diff
@@ -313,7 +313,8 @@ PTL.editor = {
     const hotkeys = mousetrap(document.body);
```

Comment:
@phlax we want to catch hotkeys even if the editor (textarea or any other control inside `#editor`) doesn't have focus. And without such focus `#editor` won't get the keyboard event.